packages/modal/src/Modal.tsx

```diff
@@ -202,14 +202,17 @@ function Modal({
         <Portal>
           <div
             {...rest}
+            role="none"
```

Comment:
What's the difference between this, and not setting `role` at all?

Reply:
Added a comment!